acrobat/scripts/alloy/verb-widget.js

```diff
@@ -23,15 +25,16 @@ if (params.dropzone2) {
   appTags.push('dropzone2');
 }
 
-export default function init(eventName, verb, metaData, documentUnloading = true) {
-  function ensureSatelliteReady(callback) {
-    // eslint-disable-next-line no-underscore-dangle
-    if (window._satellite && typeof window._satellite.track === 'function') {
-      callback();
-    } else {
-      setTimeout(() => ensureSatelliteReady(callback), 200);
-    }
+function ensureSatelliteReady(callback) {
+  // eslint-disable-next-line no-underscore-dangle
+  if (window._satellite && typeof window._satellite.track === 'function') {
```

Comment:
I would add optional chaining just in case. Alternatively you can use `instanceof`:
`if (window._satellite?.track instanceof Function) {`